vignettes/00-introduction.Rmd

```diff
@@ -1,3 +1,38 @@
 # (PART\*) Section 1: Vignettes {-}
 
-# Vignettes Introduction
\ No newline at end of file
+# Vignettes Introduction
+
+
+These vignettes provide a **quick start** tutorial - they quickly show you how to access different types of data in the TERRA Reference datasets and perform ‘hello world’ tasks with them. 
+
+The vignettes currently include examples of:
+* trait data
+* weather data
+* image data
+* combining trait, weather, and image data in a single analysis
+
+In order to make these vignettes as easy to use as possible, we are using a subset of the full dataset that is is available without an account or password. Specifically, we have made data from thirty two plots in each of the two seasons available. 
+
+Although we want to make these data broadly available, the remainder of the dataset requires an account to use so we can keep track of who and how many people are using it, and can contact users with their provided email addresses. Information for getting a beta user account is available in [section 1.1.2](https://terraref.github.io/tutorials/index.html). 
```

Comment:
```suggestion
Although we want to make these data broadly available, the remainder of the dataset requires an account to use so we can keep track of who and how many people are using it, and can contact users with their provided email addresses. 
Information for getting a beta user account is available in the section on [User Accounts](#user-accounts).
```

@KristinaRiemer could you please check that this is the correct way to use the internal referencing system described in https://bookdown.org/yihui/bookdown/cross-references.html

Reply:
@dlebauer that section reference formatting looks correct.